content/en/docs/concepts/architecture/nodes.md

```diff
@@ -111,13 +111,14 @@ This information is gathered by Kubelet from the node.
 
 ## Management
 
-Unlike [pods](/docs/concepts/workloads/pods/pod/) and [services](/docs/concepts/services-networking/service/),
-a node is not inherently created by Kubernetes: it is created externally by cloud
-providers like Google Compute Engine, or it exists in your pool of physical or virtual
-machines. So when Kubernetes creates a node, it creates
-an object that represents the node. After creation, Kubernetes
-checks whether the node is valid or not. For example, if you try to create
-a node from the following content:
+There are two main ways to have Nodes added to the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}:
```

Comment:
nit (possible edit): There are two ways to create a node with the Kubernetes API server:
1. The kubelet registers ...
1. Configure a Node object and send this object as a request to the API server [kubectl, curl]
 
nit: I think this part is missing from your addition but could be reworded:
`When Kubernetes creates a node, it creates
an object that represents the node.`

Reply:
I reworded this.